server/core/store.go

```diff
@@ -147,19 +147,24 @@ func (s *StoreInfo) IsAvailable(limitType storelimit.Type) bool {
 	return true
 }
 
-// IsUp checks if the store's state is Up.
-func (s *StoreInfo) IsUp() bool {
-	return s.GetState() == metapb.StoreState_Up
+// IsPreparing checks if the store's state is preparing.
```

Comment:
Maybe we can define a new function IsOnline or rewrite the function IsUp, which returns true if the NodeState is Serving or Preparing? Since The function `IsPreparing` and `IsServing` are used together(more than once) which is used to check if the store are online..

Reply:
LGTM

Reply:
In the future, maybe we will divide them apart. For now, I just make it work well as before. That's why I doesn't create a new function.